content/08-cardano-sidechains/03-example-evm-sidechain/04-deploy-smart-contracts/02-setup-development.mdx

```diff
@@ -0,0 +1,294 @@
+---
+title: Set up development environment
+metaTitle: Set up development environment
+---
+
+The EVM sidechain testnet is EVM-compatible with most Ethereum software to develop and deploy smart contracts including:
+
+- [Remix](#remix)
+- [Hardhat](#hardhat)
+- [Truffle](#truffle)
+
+# Remix
+
+This tutorial will show you how to deploy a Solidity smart contract to the EVM sidechain testnet using Remix. [Remix](https://remix.ethereum.org/) is an online IDE to develop and deploy Solidity smart contracts.
+
+## Prerequisites
+
+In this tutorial, you will use MetaMask as your wallet and Web3 provider. If you have not done so already, [create or import an account with MetaMask](../01-metamask.mdx) and fund your wallet with testnet tokens using the [faucet](<PENDING_URL>).
+
+## Step 1 - Configure Remix
+
+- Visit the [Remix IDE](https://remix.ethereum.org/). The left-hand side has a menu [1] and a typical file explorer [2]. The ` Solidity Compiler` [3] and `Deploy and run transactions` [4] plugins should be active by default.
+
+![remix-menu](https://user-images.githubusercontent.com/10556209/200480707-ec8b8e69-6cdc-498b-b315-c029fbc2a564.png)
+
+If the Solidity compiler and deploy plugins are not activated, you can search and activate them using the plugin manager [5] located in the bottom-left of the screen.
```

Comment:
If the Solidity compiler and deploy plugins are not activated, you can search and activate them using the plugin manager [5] located in the bottom left of the screen.